docs/cli/zetacored/cli.md

```
@@ -10330,6 +10330,7 @@ zetacored tx crosschain [flags]
 * [zetacored tx crosschain add-outbound-tracker](#zetacored-tx-crosschain-add-outbound-tracker)	 - Add an outbound tracker
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 96

(MD010, no-hard-tabs)